x-pack/plugins/security_solution/common/api/detection_engine/model/alerts/8.11.0/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ALERT_WORKFLOW_ASSIGNEE_IDS } from '@kbn/rule-data-utils';
+import type { AlertWithCommonFields800 } from '@kbn/rule-registry-plugin/common/schemas/8.0.0';
+import type {
+  Ancestor890,
+  BaseFields890,
+  EqlBuildingBlockFields890,
+  EqlShellFields890,
+  NewTermsFields890,
+} from '../8.9.0';
+
+/* DO NOT MODIFY THIS SCHEMA TO ADD NEW FIELDS. These types represent the alerts that shipped in 8.11.0.
+Any changes to these types should be bug fixes so the types more accurately represent the alerts from 8.11.0.
+If you are adding new fields for a new release of Kibana, create a new sibling folder to this one
+for the version to be released and add the field(s) to the schema in that folder.
+Then, update `../index.ts` to import from the new folder that has the latest schemas, add the
+new schemas to the union of all alert schemas, and re-export the new schemas as the `*Latest` schemas.
+*/
+
+export type { Ancestor890 as Ancestor8110 };
+
+export interface BaseFields8110 extends BaseFields890 {
+  [ALERT_WORKFLOW_ASSIGNEE_IDS]: string[] | undefined;
```

Comment:
```suggestion
  [ALERT_WORKFLOW_ASSIGNEE_IDS]?: string[];
```

Reply:
I just wanted to follow the pattern in all other `alerts/8.x.0/index.ts` files, so decided to leave `undefined` here

Reply:
For the alert schema `: string[] | undefined` is preferable to `?: string[]` because it helps to prevent cases where we might forget to handle the field, e.g. if since we build alert documents in a few different ways this ensures that we handle `ALERT_WORKFLOW_ASSIGNEE` in all of them instead of accidentally forgetting some places.